test/mpi/errors/datatype/testlist

```diff
@@ -1 +1,10 @@
-getcnterr 1
+getcnterr 1 xfail=ticket2697
```

Comment:
Can you make this change to xfail `getcnterr` a separate commit? It is unrelated to the new tests being added in this commit.